Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityDeviceManager.cs

```diff
@@ -320,10 +320,7 @@ public override void Enable()
 
             if (InputSystemProfile == null) { return; }
 
-            IMixedRealityInputSystem inputSystem = Service as IMixedRealityInputSystem;
-
-            if ((inputSystem != null) &&
```

Comment:
Can you better emphasize in description that changes here extend from your other PR. Like add **NOTE** just to be extra clear for readers

Reply:
I'd also add that this is dependent on that PR being checked in first....so then it'll be easier for people to review what this PR change really entails

Reply:
I did the in the Overview description above "Based on #6779". Wasn't clear then, sorry?

Reply:
Yes, sorry let me elaborate. I saw the note. My comment here is to just make it stand out more and give more information. like the following. This is a nit thing, it just helps when reviewing

**IMPORTANT** : This PR extends from changes in PR blah and should be checked in after

Reply:
Ah yes, sorry, didn't get the "emphasize" on first read... :)